test/FunctionalTests/Microsoft.OData.Edm.Tests/Vocabularies/ValidationVocabularyTests.cs

```diff
@@ -86,7 +86,7 @@ public void TestValidationVocabularyModel()
   <Term Name=""AllowedValues"" Type=""Collection(Validation.AllowedValue)"" AppliesTo=""Property Parameter TypeDefinition"" Nullable=""false"">
     <Annotation Term=""Core.Description"" String=""A collection of valid values for the annotated property, parameter, or type definition"" />
   </Term>
-  <Term Name=""MultipleOf"" Type=""Edm.Decimal"" AppliesTo=""Property Parameter Term"" Nullable=""false"" Scale=""Variable"">
+  <Term Name=""MultipleOf"" Type=""Edm.Decimal"" AppliesTo=""Property Parameter Term"" Nullable=""false"" Scale=""variable"">

```

Comment:
@ElizabethOkerio @wachugamaina It's "variable" in the OData vocabularies repo https://github.com/oasis-tcs/odata-vocabularies/blob/9596c6a66f992ce7f33f950976a252e945bc91bb/vocabularies/Org.OData.Validation.V1.xml#L108.
But since we're adding a feature flag to allow someone to toggle the old behaviour, we should have a test to validate that if the flag is set, "Variable" is handled correctly.
It doesn't have to be this one for validation the OData validation vocabulary - it can be a much simpler test verifying the behavior against `Scale="Variable"` and `Scale="variable"`.
If such a test already exists then it's fine

Reply:
There are tests for this.